src/IO/S3/PocoHTTPClient.cpp

```diff
@@ -62,6 +79,46 @@ void PocoHTTPClient::MakeRequestInternal(
     auto uri = request.GetUri().GetURIString();
     LOG_DEBUG(log, "Make request to: {}", uri);
 
+    enum class ES3MetricType
```

Comment:
According to our style guide, the `E` prefix should not be used.

Reply:
Fixed